### PR TITLE
made the banner smaller

### PR DESCRIPTION
**Pull request by Kelvinrr**
Minor update to the ISIS docs to reduce the size of the banner and add a link to the new docs. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added my user impacting change to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document. 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/docsys/build/homepage.xml [hunk 1]
```diff
@@ -29,4 +29,10 @@
     Reconnaissance Orbiter, MESSENGER, Lunar Reconnaissance Orbiter,
     Chandrayaan, Dawn, Kaguya, and New Horizons.
   </p>
+  <h2><a name=">AstrogeologySupportingDocumentation">Astrogeology Supporting Documentation</a></h2>
+  <p>
+    Supporting ISIS Documentation containing tutorials, how-to guides and explanations of how to use the software 
+    has been moved to <a href="https://astrogeology.usgs.gov/docs/">https://astrogeology.usgs.gov/docs/</a>. Previous docs 
+    for older versions of ISIS can be accessed on the left navigation pane.
```
Comment (Kelvinrr) [inline isis/src/docsys/build/homepage.xml]: I feel there is a higher visibility way to do this?

**Comment by amystamile-usgs** [comment]
I don't really see the Moon image in the banner anymore. Would it be worth using a different image or resizing? Just looks black.

**Comment by jrcain-usgs** [comment]
The bigness is sort of the point, if you're going to make it as much smaller as this commit does, you might as well just remove it.

The point of the banner is to have a big image/open space on the homepage, something other than text that's hopefully nice to look at, and aligned with the idea of ISIS.  Its value is mostly aesthetic, and It loses this if you make it smaller.  I think we should leave it big?  But if we can't we should just remove it.

File: isis/src/docsys/build/homepage.xml (hunk 1 above)
Comment (jrcain-usgs) [reply isis/src/docsys/build/homepage.xml]: Something between [USWDS Cards](https://designsystem.digital.gov/components/card/) and the tiles on the new ISIS Applications by Category page?

Comment (jrcain-usgs) [reply isis/src/docsys/build/homepage.xml]: A bigger banner can also be a place for info/news like this

Comment (jrcain-usgs) [reply isis/src/docsys/build/homepage.xml]: Screenshot 2024-11-12 at 14 46 18

**Comment by Kelvinrr** [comment]
@jrcain-usgs Im fine with removing it, I feel the original was way too big and kinda got in the way. If we wanted a big banner we should go something flashier e.g. https://squidfunk.github.io/mkdocs-material/

File: isis/src/docsys/build/homepage.xml (hunk 1 above)
Comment (jrcain-usgs) [reply isis/src/docsys/build/homepage.xml]:
```html
<style>

    /* Should go to isisStyleCommons.css, replace current #isis-banner style */

    #isis-banner {
        background: 
        linear-gradient(
            rgba(0, 0, 0, 0.2), 
            rgba(0, 0, 0, 0.3)
            ),
        url(../img/moon-cr.jpg);
        color: white;
        padding: 4rem;
        background-size: cover;
        display: flex;
        flex-direction: row;
        flex-wrap: wrap;
        justify-content: space-between;
    }

    .news-card {
        margin: 18px 30px;
        padding: 30px;
        border: 3px solid darkslategray;
        border-radius: 5px;
        width: 28rem;
    }

    .news-card a {
        color: white;
    }

    /* Probably should add the usa-button style too */
</style>


<div id="isis-banner">  <!--replaces current id="isis-banner" div-->

    <div>
        <div class="isis-logo"></div>
        <p>
        <em>Integrated Software for Imagers and Spectrometers</em>
        </p>
    </div>
        
    <div class="news-card">
        <h2>
        Some Docs have Moved
        </h2>
        <p>
        Manuals for each ISIS application <a href="#TODO: link to Applications By Category">remain here</a>, but tutorials and guides have been moved to ASC Docs.
        </p>
        <button>
        Visit ASC Docs
        </button>
    </div>

</div>


```

**Comment by jrcain-usgs** [comment]
I don't have anything that flashy, but I do have this new ISIS logo if you want, because the text in the old one was not that readable

**Comment by jrcain-usgs** [comment]
The banner image I got from [Project Apollo Archive](https://www.flickr.com/photos/projectapolloarchive/), because those are public domain.  They have some other reasonable images, but if you host the image outside of the git repo, you can use something that's not public domain.

Here's one from NASA I liked, NASA photos/art aren't necessarily in the public domain though: [Chandra Spacecraft in Orbit](https://photojournal.jpl.nasa.gov/catalog/PIA18166)

**Review by amystamile-usgs — APPROVED** [review]

**Pull request merged**